packages/core/package.json

```diff
@@ -18,10 +18,9 @@
     ],
     "scripts": {
         "build": "yarn build:styles && yarn build:modules",
-        "build:css": "node-sass src -o build/css --importer='../../node_modules/node-sass-tilde-importer'",
+        "build:styles": "node-sass . -o . --non-recursive --importer='../../node_modules/node-sass-tilde-importer'",
         "prebuild:scss": "mkdirp -p build && mkdirp build/scss",
```

Comment:
This can go

Reply:
Yep! That was me not remembering the underscore thing in Sass 😅

Reply:
Yep! And `--non-recursive` can be removed as well